sdk/test-utils/recorder/test/testProxyClient.spec.ts

```diff
@@ -8,16 +8,18 @@ import {
   PipelineResponse,
 } from "@azure/core-rest-pipeline";
 import { expect } from "chai";
-import { env, Recorder } from "../src";
-import { createRecordingRequest } from "../src/utils/createRecordingRequest";
-import { paths } from "../src/utils/paths";
-import { getTestMode, isLiveMode, isRecordMode, RecorderError } from "../src/utils/utils";
+import { Recorder } from "../src/index.js";
+import { createRecordingRequest } from "../src/utils/createRecordingRequest.js";
+import { paths } from "../src/utils/paths.js";
+import { getTestMode, isLiveMode, isRecordMode, RecorderError } from "../src/utils/utils.js";
+import { describe, it, beforeEach, afterEach, TaskContext } from "vitest";
```

Comment:
`TaskContext` should be a type and referenced as `type TaskContext`

Reply:
updated